changelog.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v0.4.0 | 2019-12-05
+
+* Add calibration data as attribute of force channels.
```

Comment:
It would be good to include an example in the tutorial.

Reply:
Added.